### ISSUE TITLE
Test Results Node - MVP1 - No Description

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#429]
Looks like we lost the result node description on MVP1 results.  The descriptions are showing for MVP2 results, but I'm not seeing them on What drugs may treat (in Test 7/14):

[Sneddon Syndrome](https://ui.test.transltr.io/results?l=Sneddon%20Syndrome&i=MONDO:0008436&t=0&q=e8c32220-a5e2-4985-9af7-0b25ffc702fc):

<img width="863" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/3afedf26-df0e-426b-bd8c-6d32ff9dfa24">

[Meniere Disease](https://ui.test.transltr.io/results?l=Meniere%20Disease&i=MONDO:0007972&t=0&q=eec1a17a-1bd1-44c2-9e81-5e7e4b350f3c)
<img width="939" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/6f406214-6c8d-4360-aeb5-b2920927a107">

[Familial Pityriasis Rubra Pilaris](https://ui.test.transltr.io/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&q=348edc45-9110-460b-baf0-ea784acc4f9c)
<img width="888" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/daf65a7f-c340-4b2f-8c00-383fa859a845">

May be related to #112

**Comment by dnsmith124** [comment]
I don't see any issue with the data on the FE in these examples. 

This is likely an attribute server issue, but I'll let @gprice1129 confirm.

**Comment by newgene** [comment]
Yes, pending on this MyChem.info issue we are working on right now: https://github.com/biothings/mychem.info/issues/159

**Comment by sstemann** [comment]
@newgene will this be resolved by the 8/18 code freeze?

**Comment by newgene** [comment]
@sstemann yes, that's our plan.

**Comment by newgene** [comment]
@sstemann just fyi we have completed this dependency issue at https://github.com/biothings/mychem.info/issues/159. @MarkDWilliams and @Genomewide's teams are working on the integration with the ARS and UI.

**Comment by sstemann** [comment]
@newgene i'm not sure if i'm testing this right but i have results from the Prod UI for MVP1 [Maturity-onset Diabetes Of The Young](http://purl.obolibrary.org/obo/MONDO_0018911)

https://ui.transltr.io/main/results?l=Maturity-onset%20Diabetes%20Of%20The%20Young&i=MONDO:0018911&t=0&q=4231e8a1-4ae4-4f4f-9b5e-53774698655b

and some of the tops results dont have descriptions, if I'm doing this right in annotator, then I'm guessing its because its not available?

Insulin detemir - No description in the UI
<img width="860" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/daf288ea-06ce-4602-bb7e-fae9028c416c">

UI Links to https://pubchem.ncbi.nlm.nih.gov/compound/16137271
No definition in the annotator: https://biothings.ncats.io/annotator/PUBCHEM.COMPOUND:16137271?fields=unii.ncit,chebi.definition
<img width="425" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/20fba27e-b006-4eee-a3fa-945590a4f1e6">

Then if i find one that's Chebi like Uric Acid
<img width="848" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/0a05d46b-c45e-44cf-ad1c-a7d7324ecfd6">

There's no description, but here's what the UI LInks out to:
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27226
And what the annotator has on it
https://biothings.ncats.io/annotator/CHEBI:27226?fields=unii.ncit,chebi.definition
<img width="811" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/63d5f93c-359d-42ba-9622-58518ac0f11c">

But that definition "An oxopurine that is the final oxidation product of purine metabolism." isn't showing up in the UI.

So it looks like there are some substances without descriptions and some with definitions that aren't in the UI, is that right?

Is this by design?

**Comment by newgene** [comment]
> UI Links to https://pubchem.ncbi.nlm.nih.gov/compound/16137271
> No definition in the annotator: https://biothings.ncats.io/annotator/PUBCHEM.COMPOUND:16137271?fields=unii.ncit,chebi.definition

I checked the above case, looks like the root reason is because the record of `PUBCHEM.COMPOUND:16137271` in MyChem.info is not linked with the record of `UNII:4FT78T86XV`. That unii record includes the mapping to NCIT id of `C65920`, which allows us to get the drug description from NCIT. We will take a closer look of this issue at MyChem.info side at https://github.com/biothings/mychem.info/issues/169.

**Comment by gprice1129** [comment]
The definition of done for this ticket is extremely unclear. I suggest we close this ticket and open up "No Description" tickets for individual results/nodes.

**Comment by Genomewide** [comment]
@newgene, how often do we expect to have a description, and when should we expect not to have a description?

**Comment by newgene** [comment]
@Genomewide @gprice1129 this issue has been fixed upstream in the first new release of MyChem.info in 2024 :-)

This issue https://github.com/biothings/mychem.info/issues/169 has been closed.

I have verified NCIT description now returned from the annotator:

https://biothings.ncats.io/annotator/CHEBI:27226?fields=unii.ncit,chebi.definition

**Comment by sstemann** [comment]
i get a little confused - there are still no descriptions sometimes.  in TEST UI:

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/61244a5b-6ff6-470e-99fe-17e50916f10f)

https://ui.test.transltr.io/main/results?l=Ehlers-danlos%20Syndrome,%20Vascular%20Type&i=MONDO:0017314&t=0&r=0&q=4b7cd9bb-3154-4257-867f-ffcb26d23d6c

@Genomewide

**Comment by gprice1129** [comment]
@sstemann we will be looking into descriptions this cycle.

**Comment by sstemann** [comment]
@gprice1129 i think on 3/27 release their were some bullets about annotation server just noting here i'm still seeing sparsity on these descriptions:

https://ui.test.transltr.io/main/results?l=Exotropia&i=MONDO:0001286&t=0&r=0&q=0b2a500e-a935-4191-9f39-7ade0e25ce83

<img width="940" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/f633effe-ff71-433c-9c82-63a5ccd5bf56">


https://ui.test.transltr.io/main/results?l=Apidra&i=PUBCHEM.COMPOUND:16136701&t=3&r=0&q=73bce986-35eb-4bb3-92f3-e2fc28e7464e

<img width="946" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/be45c5bb-8864-49e1-9fac-66c624eab900">

**Comment by gprice1129** [comment]
@sstemann this is on our radar but was not part of the annotation work we did.

**Comment by gaurav** [comment]
This probably won't be super helpful here, since I think you're mainly talking about genes and chemicals which we don't have descriptions for yet, but NodeNorm now returns descriptions for CURIEs in Ubergraph, such as [UBERON:0003684](https://nodenorm.transltr.io/get_normalized_nodes?curie=UBERON%3A0003684&conflate=true&drug_chemical_conflate=false&description=true):

```json
{
  "UBERON:0003684": {
    "id": {
      "identifier": "UBERON:0003684",
      "label": "abdominal cavity",
      "description": "The part of the ventral body cavity that is within the abdomen proper (excluding the pelvic cavity)."
    }, ...
  }
}
```

We only get our descriptions from Ubergraph at the moment, so only ontology CURIEs can have descriptions. But if there's a source of descriptions we can ingest we'd be happy to do that.

**Comment by sstemann** [comment]
still a problem:  https://ui.test.transltr.io/main/results?l=Sneddon%20Syndrome&i=MONDO:0008436&t=0&r=0&q=80e2fc7c-f372-4438-ba0e-82707af60b15

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/9dc472f9-64aa-4aa5-a7b9-c7b8f44d83f9)

**Comment by sstemann** [comment]
still no descriptions on thi MVP1 : https://ui.test.transltr.io/main/results?l=Sneddon%20Syndrome&i=MONDO:0008436&t=0&r=0&q=05421421-a30e-4f28-8e5c-8a0312be0e2a

**Comment by sierra-moxon** [comment]
@sstemann - I believe this is fixed or otherwise I can not find results without descriptions in the first several pages of returns today for the query above.  

I did notice a discrepancy in the view w/re to descriptions though, and its not clear to me why?  -- for example:
Verapamil has a description without opening the results, but the other two chemicals in this result appear to not have descriptions:
<img width="1229" alt="Screen Shot 2024-07-11 at 11 50 15 AM" src="https://github.com/NCATSTranslator/Feedback/assets/4561292/a5344646-5714-4a88-8476-e0ffff4f7e1f">

however, when I "open" the results without the descriptions, I do see the descriptions after all:

<img width="1266" alt="Screen Shot 2024-07-11 at 11 50 29 AM" src="https://github.com/NCATSTranslator/Feedback/assets/4561292/0db1dd69-9723-4c35-b2b5-5cc943f18f40">

<img width="1226" alt="Screen Shot 2024-07-11 at 11 50 40 AM" src="https://github.com/NCATSTranslator/Feedback/assets/4561292/3a51c90e-afbf-4754-8a4c-691ba9a2a515">

Is this the issue we're trying to solve or can we close this as complete?

**Comment by dnsmith124** [comment]
@sierra-moxon Based on user feedback the UI team determined that the chebi role tags on results are more important than the description, so if any are present they bump the description down "below the fold" so to speak. 

That's not the issue this ticket was concerned with though, so I believe we can close!
